versions/3.0.4.md

```diff
@@ -130,7 +132,7 @@ It is RECOMMENDED that the entry OpenAPI document be named: `openapi.json` or `o
 
 When parsing an OAD, JSON or YAML objects are parsed into specific Objects (such as [Operation Objects](#operation-object), [Response Objects](#response-object), [Reference Objects](#reference-object), etc.) based on the parsing context. Depending on how references are arranged, a given JSON or YAML object can be parsed in multiple different contexts:
 
-* As a complete OpenAPI Description document
+* As a syntactically complete OpenAPI Description document
```

Comment:
When do we ever parse a document in the context of "a (syntactically) complete OAD" rather than one of the other options?

Reply:
It is the objects that are parsed "in the context", not documents.  If you have a syntactically complete document, then you know that the root object is an OpenAPI Object.

You can't parse the root object of the entry document by either parent context or reference context because neither of those contexts exist.  But you can only parse a syntactically complete document without those contexts.

Basically, if you just get some random document without any other context, you _MUST_ treat it as an entry document, and error if the root object is not an OpenAPI Object.  It might be better to state that more directly and reserve the "context" wording for the other two cases?

Reply:
Now I understand. I think you are saying that if there is no parent object, then the context is an OpenAPI description. Let me think about how this could be more clear.